tests/models/pvnet/test_utils.py

```diff
@@ -0,0 +1,28 @@
+""" Tests for utils for pvnet"""
+import numpy as np
+from ocf_datapipes.batch import BatchKey
+
+from india_forecast_app.models.pvnet.utils import set_night_time_zeros
+
+
+def test_set_night_time_zeros():
+    """ Test for setting night time zeros"""
+    # set up preds (1,14,7)
```

Comment:
Small nit, is this comment slightly out of date?

Reply:
oh yes